Dockerfile

```diff
@@ -23,22 +25,27 @@ RUN apt-get update && apt-get install -y \
     \
     less \
     man \
-    vim
+    vim \
+    \
+    libssl-dev
+
+COPY dist/brainiak-* /mnt/brainiak
+
+WORKDIR /mnt/brainiak
 
-WORKDIR /mnt
+COPY tutorials/tutorials tutorials
 
 RUN set -e \
-    && python3 -m pip install --user -U pip \
-    && python3 -m pip install -U brainiak \
-    && python3 -m pip download --no-deps --no-binary :all: brainiak \
-    && export BRAINIAK_VERSION=$(basename brainiak-* .tar.gz | cut -b 10-) \
-    && tar -xf brainiak-*.tar.gz \
-    && for example in brainiak-$BRAINIAK_VERSION/examples/*/requirements.txt; \
+    && python3 -m pip install --user -U "pip<10" \
+    && python3 -m pip install --user -U . \
+    && python3 -m pip install --user -U -r tutorials/requirements.txt \
+    && for example in examples/*/requirements.txt; \
         do python3 -m pip install --user -U -r $example ; done
 
 RUN echo PATH=\"\$HOME/.local/bin:\$PATH\" >> $HOME/.profile \
     && echo "shell -bash" >> ~/.screenrc
 
-EXPOSE 8888
+EXPOSE 8899
 
 ENTRYPOINT ["/bin/bash", "-l"]
+CMD ["tutorials/run_jupyter_docker.sh"]
```

Comment:
Do we want the tutorials to automatically start? Seems like most of the time the user of brainiak docker will not be using the tutorials.

Reply:
I think the tutorials are a great way to start for all users of BrainIAK. At the same time, we should add instructions to the website about starting a shell (hint: add `-c bash` at the end of the `docker run` command).